onchaind/onchain_wire.csv

```diff
@@ -48,6 +48,7 @@ onchain_init_reply,5101
 # onchaind->master: Send out a tx.
 onchain_broadcast_tx,5003
 onchain_broadcast_tx,,tx,struct bitcoin_tx
+onchain_broadcast_tx,,type,u16
```

Comment:
Use enum here, and define towire_wallet_txtype and fromwire_ somewhere?

This means moving wallet_txtypes out to common; maybe just call it 'txtype' then?